src/frontend/src/modals/apiModal/utils/get-js-api-code.tsx

```diff
@@ -18,29 +18,37 @@ export default function getJsApiCode({
 }: GetCodeType): string {
   let tweaksString = "{}";
   if (tweaksBuildedObject)
-    tweaksString = JSON.stringify(tweaksBuildedObject, null, 8);
+    tweaksString = JSON.stringify(tweaksBuildedObject, null, 2)
+      .replace(/^ {2}/gm, "    ")
+      .replace(/}$/, "  }");
   const inputs = useFlowStore.getState().inputs;
   const outputs = useFlowStore.getState().outputs;
   const hasChatInput = inputs.some((input) => input.type === "ChatInput");
   const hasChatOutput = outputs.some((output) => output.type === "ChatOutput");
 
-  return `${activeTweaks ? "" : 'let inputValue = ""; // Insert input value here\n\n'}fetch(
-  "${window.location.protocol}//${window.location.host}/api/v1/run/${endpointName || flowId}?stream=false",
-  {
-    method: "POST",
-    headers: {
-      "Authorization": "Bearer <TOKEN>",
-      "Content-Type": "application/json",${isAuth ? '\n\t\t\t"x-api-key": <your api key>' : ""}
-    },
-    body: JSON.stringify({${activeTweaks ? "" : "\n\t\t\tinput_value: inputValue, "}
-      output_type: ${hasChatOutput ? '"chat"' : '"text"'},
-      input_type: ${hasChatInput ? '"chat"' : '"text"'},
-      tweaks: ${tweaksString}
-    }),
-  },
-)
-  .then(res => res.json())
-  .then(data => console.log(data))
-  .catch(error => console.error('Error:', error));
-`;
+  return `# Install the Langflow client
+# npm install @datastax/langflow-client
+
+import { LangflowClient } from "@datastax/langflow-client";
+
+const client = new LangflowClient({
+  baseUrl: "${window.location.protocol}//${window.location.host}"${isAuth ? ",\n  apiKey: <your api key>" : ""}
+})
+const flow = client.flow("${endpointName || flowId}");
+
+let inputValue = ""; // Insert input value here
+
+flow.run(inputValue, {
+  output_type: ${hasChatOutput ? '"chat"' : '"text"'},
+  input_type: ${hasChatInput ? '"chat"' : '"text"'},
+  session_id: "user_1",
+  tweaks: ${tweaksString}
+}).then(response => {
+  // get the text of the first chat response
+  console.log(response.chatOutputText());
+  // Or get all the outputs
+  console.log(response.outputs);
+}.catch(error => {
+  console.error(error);
+});`;
```

Comment:
Just a small suggestion since `#` isn't valid JS.

```suggestion
  return `import { LangflowClient } from "@datastax/langflow-client"; // be sure to \`npm install @datastax/langflow-client\`

const client = new LangflowClient({
  baseUrl: "${window.location.protocol}//${window.location.host}"${isAuth ? ",\n  apiKey: <your api key>" : ""}
})
const flow = client.flow("${endpointName || flowId}");

let inputValue = ""; // 👈 Insert your input value here

flow.run(inputValue, {
  output_type: ${hasChatOutput ? '"chat"' : '"text"'},
  input_type: ${hasChatInput ? '"chat"' : '"text"'},
  session_id: "user_1",
  tweaks: ${tweaksString}
}).then(response => {
  // get the text of the first chat response
  console.log(response.chatOutputText());
  // Or get all the outputs
  console.log(response.outputs);
}.catch(error => {
  console.error(error);
});`;
```

Reply:
🤦‍♂️ I know how JS works, I promise 😭